webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -255,7 +307,38 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
         leftContent={(
           <div className={css.base}>
             <div className={css.experimentInfo}>
-              <ExperimentState experiment={experiment} />
+              <Spinner spinning={isChangingState}>
+                <div className={css.experimentState} style={{ backgroundColor }}>
+                  {experiment.state === RunState.Active && (
```

Comment:
question: do we need to consider job states? maybe using `pausableStates`

Reply:
ah yeah I added them when I rebased with your job states work.